### ISSUE TITLE
Error: Uncaught BadMethodCallException when viewing block archives when a descendant of ElementList has been archived

### ISSUE THREAD

**Issue by ChrissiQ** [silverstripe/silverstripe-versioned-admin#118]
Relevant package versions:
SilverStripe Versioned 1.4.1
SilverStripe Versioned Admin 1.2.1
SilverStripe Framework 4.4.1
Elemental 4.1
Elemental List 1.2

On a site I am developing, when viewing the block archive page (/admin/archive/DNADesign-Elemental-Models-BaseElement), the page breaks with an error:

```
[Emergency] Uncaught BadMethodCallException: Object->__call(): the method 'Breadcrumbs' does not exist on '[...descendant of ElementList...]'

Trace
SilverStripe\View\ViewableData->__call(Breadcrumbs, Array) 
BlockArchiveExtension.php:51
```

I believe the block's parent is being assumed to be a page, rather than another element, and thus the `Breadcrumbs` method doesn't exist on that element when it's an ElementList instead.

If I have time I might try to submit a PR for this but not sure if I have time, I'd appreciate someone else looking into this.

**Comment by ScopeyNZ** [comment]
> I believe the block's parent is being assumed to be a page, rather than another element, and thus the Breadcrumbs method doesn't exist on that element when it's an ElementList instead.

Hmmn this sounds like you're on the right track. This issue might be fixed in elemental if that's the case. I'll leave this open here until it can be verified.

**Comment by a2nt** [comment]
I think versioned-admin has to check if parent object has method Breadcrumbs

**Comment by robbieaverill** [comment]
Fixed in #168
